modules/ytp-assets-common/src/less/components/list--dataset.less

```diff
@@ -60,3 +67,32 @@
     }
   }
 }
+
+.float-clearer {
+  clear: both;
+}
+
+.dataset-title {
+  float: left;
+}
+
+.private,
+.draft,
+.deleted {
+  float: right;
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 14px;
+  font-family: @font-family-sans-serif-semibold;
+  font-weight: @font-weight-heavier;
+  text-align: center;
+  line-height: 18px;
+  background: #00347a;
+  border: 1px solid #00347a;
```

Comment:
`@suomi-brandBase` is so close to this color, that my guess is the color from design and existing variable are meant to be the same